src/Pipedrive.net/Models/Response/DealActivity.cs

```diff
@@ -27,7 +27,7 @@ public class DealActivity
         public string ReferenceId { get; set; }
 
         [JsonProperty("due_date")]
-        public DateTime? DueDate { get; set; }
+        public string DueDate { get; set; }
```

Comment:
Open a specific issue for that. I'll take a look, don't change that here.

Reply:
Okay. Should I at least place a comment block here referencing the possible issue?